install/kubernetes/cilium/templates/cilium-envoy/daemonset.yaml

```diff
@@ -75,11 +75,8 @@ spec:
           httpGet:
             host: {{ .Values.ipv4.enabled | ternary "127.0.0.1" "::1" | quote }}
             path: /healthz
-            port: {{ .Values.envoy.healthPort }}
+            port: {{ .Values.envoy.prometheus.port }}
```

Comment:
this implied prometheus.enabled must be true, however, it seems like we always need health check port.

Reply:
@sayboras 

thanks - yes that's irritating. 

re-used the `healthPort` when implemented the dedicated envoy listener for health aspects - as mentioned in [above comment](https://github.com/cilium/cilium/pull/25081#discussion_r1180634222).